src/App.css

```diff
@@ -1,65 +1,305 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+body {
+  background-color: #fafafa;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  box-shadow: 0 0 1.5px 0 rgba(0, 0, 0, 0.25);
+  padding: 0.8rem 6.25rem;
+}
+
+.user-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 30px;
+  height: 30px;
+  border-radius: 5rem;
+  box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.514);
+  cursor: pointer;
+}
+
+header img {
+  width: 15px;
+}
+
 .App-header {
   display: flex;
   align-items: center;
   gap: 2rem;
-  font-size: 1rem;
+}
+
+.App-header h1 {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
   color: #0290ff;
-  margin: 0.1rem 2rem;
 }
 
 .nav {
   display: flex;
+  padding: 0;
   gap: 2rem;
   list-style: none;
 }
 
 .nav-text {
-  color: rgb(62, 62, 62);
+  width: 3.688rem;
+  height: 1rem;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
   text-decoration: none;
 }
 
-.books-container {
-  background-color: #f5f6fa;
-  padding: 2rem 0 3rem;
-  height: 100%;
-}
-
 .books {
   list-style: none;
+  padding: 0.5rem 0;
 }
 
-.book-title {
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 1.5rem;
-  font-weight: 500;
+.book-div {
+  display: grid;
+  grid-template-columns: 3fr 1fr 1fr;
+  align-items: center;
+  background-color: white;
+  padding: 0.5rem 0;
+  margin: 1rem 6.25rem;
+  box-shadow: 0 0 1.5px 0 rgba(0, 0, 0, 0.25);
 }
 
 .book-details {
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 1rem;
-  font-weight: 400;
-  padding: 0 1rem 0 0;
+  padding-left: 1.688rem;
 }
 
-.remove-button {
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 1rem;
-  font-weight: 400;
+.category {
+  height: 1.125rem;
+  margin-bottom: 0.2rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title {
+  width: 12.313rem;
+  height: 1.813rem;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
```

Comment:
- [ ] There are some repetitive styles in the code, such as font definitions. You can create CSS variables for commonly used properties to improve maintainability and follow CSS best practices.🤔🤔
- [ ] See below as an example:
```css
:root {
  --pale-gray: #f5f6fa;
  --black-two: #121212;
  --black-5: rgba(0, 0, 0, 0.05);
  --medium-blue: #307bbe;
  --cool-blue: #4386bf;
  --azure: #0290ff;
  --z: #fafafa;
  --white-two: #fff;
  --very-light-pink: #c4c4c4;
  --font-family-robo: "Roboto Slab", serif;
  --font-family-monts: "Montserrat", sans-serif;
}
```